dev-libs/jansson/jansson-2.9.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit autotools multilib-minimal
+
+DESCRIPTION="C library for encoding, decoding and manipulating JSON data"
+HOMEPAGE="http://www.digip.org/jansson/"
+SRC_URI="http://www.digip.org/jansson/releases/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
+IUSE="doc static-libs"
+
+DEPEND="doc? ( >=dev-python/sphinx-1.0.4 )"
+RDEPEND=""
+
+src_prepare() {
+	default
+	sed -ie 's/-Werror//' src/Makefile.am || die
+	eautoreconf
+}
+
+multilib_src_configure() {
+	ECONF_SOURCE="${S}" \
+	econf \
+		$(use_enable static-libs static)
+}
+
+multilib_src_compile() {
+	default
+
+	if multilib_is_native_abi && use doc ; then
+		emake html
+	fi
```

Comment:
this one's not important, calling `default` in `multilib_src_install_all` is wrong though